contribute/style-guides/documentation-style-guide.md

```diff
@@ -315,6 +326,10 @@ Two words if used as a verb, one word if used as a noun. Do not use _plug-in_.
 - Plug in the appliance.
 - Download the plugin.
 
+#### repository
```

Comment:
@chri2547, this is already in our linter at https://github.com/grafana/technical-documentation/blob/main/linters/vale/Grafana/WordList.yml#L68.